<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grupy1</class>
 <widget class="QWidget" name="Grupy1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grupy1</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregue uma ou mais imagens</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_selected_images">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="load_OK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregado</string>
   </property>
  </widget>
  <widget class="QListWidget" name="codes_list">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </widget>
  <widget class="QLabel" name="load_OK_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="processButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Processar</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
